sys/include/net/gnrc/ipv6/nib/conf.h

```diff
@@ -21,59 +21,61 @@
 #ifndef NET_GNRC_IPV6_NIB_CONF_H
 #define NET_GNRC_IPV6_NIB_CONF_H
 
+#include <kernel_defines.h>
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 /* some pseudo-module based configuration, doc: see below */
 #ifdef MODULE_GNRC_IPV6_NIB_6LBR
-#ifndef GNRC_IPV6_NIB_CONF_6LBR
-#define GNRC_IPV6_NIB_CONF_6LBR         1
+#ifndef CONFIG_GNRC_IPV6_NIB_CONF_6LBR
+#define CONFIG_GNRC_IPV6_NIB_CONF_6LBR      1
```

Comment:
To all macro names of this pattern: these names are pretty redundant. Why not use e.g. `CONFIG_GNRC_IPV6_NIB_6LBR`?

Reply:
Done, I removed the inner `_CONF`.